docs/topics/Voice_Connections.md

```diff
@@ -234,6 +234,9 @@ The following flags can be used as a bitwise mask. For example `5` would be prio
 > warn
 > You must send at least one [Opcode 5 Speaking](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/voice) payload before sending voice data, or you will be disconnected with an invalid SSRC error.
 
+> info
+> The `delay` property is ignored for voice connections via the voice gateway.
+
```

Comment:
Not true and it is very important. For browser, it must be set to VAD leading value. For Discord Client, it must be set to zero.

Reply:
This is only usable by bots though, and bots can't be used in a browser context due to CORS. I'm not sure what I should write then?

Reply:
Sounds good.